### ISSUE TITLE
Read from VSAC

### ISSUE THREAD

**Issue by Sigfried** [jhu-bids/TermHub#130]
Need to clarify and decide on how to prioritize.

Stephanie: target to SSSOM format.

TermHub handle SSSOM? How soon?

**Comment by joeflack4** [comment]
From what I understand on the SSSOM side, it involves Stephanie creating SSSOM files and then using `sssom-py` to transform to FHIR ConceptMap JSON.

We can configure TermHub to read from the VSAC APIs. I think what people were suggesting is that we can use our concept set comparison tool to check differences for a single value set between VSAC's normal API and FHIR API.

**Comment by joeflack4** [comment]
Harold'9/28:
> Use the diffing tool to show differences between VSAC standard and FHIR APIs.

**Comment by hugwuoke** [comment]
Still relevant?

**Comment by joeflack4** [comment]
@hugwuoke Yep. Similar to the `ConceptMap` issues, this is low urgency and not planned for the immediate future. I'd say this will happen sooner than that, though, but also maybe 6-12 months or so.
